docs/pages/base-ui.tsx

```diff
@@ -16,10 +16,10 @@ export default function Core() {
   return (
     <BrandingCssVarsProvider>
       <Head
-        title="Base UI: Ship accessible & sleek components"
+        title="Base UI: Unstyled React components and low-level hooks."
```

Comment:
Oopsy 😄

```suggestion
        title="Base UI: Unstyled React components and low-level hooks"
```

<img width="705" alt="SCR-20240404-tlel" src="https://github.com/mui/material-ui/assets/3165635/fb2f7719-0b58-4924-8ae2-4b321e2ef65d">

Fixed in 26e6c2f258e4d92b010f508f200a020db8f8e895.